docs/hacking/selfmedicate/selfmedicate.rst

```diff
@@ -0,0 +1,124 @@
+About Self-Medicate
+===================
+
```

Comment:
Here's where I am thinking we could have an introductory section that points to the vagrant page as a "primary" way of running selfmedicate (which takes care a lot of the commands below as well), but also offer an explanation that if you're running linux, you can run the script directory and follow the instructions below.

Reply:
These changes have been made.